aplt/runner.py

```diff
@@ -335,6 +355,18 @@ def parse_statsd_args(args):
         return metrics.SinkMetrics()
 
 
+def group_kw_args(argList=[]):
```

Comment:
Why not *args? Putting mutable defaults like [] or {} in the func def is frowned on, because the func def is evaluated just once. This means repeat calls get the previously existing mutable list if argList is not supplied.